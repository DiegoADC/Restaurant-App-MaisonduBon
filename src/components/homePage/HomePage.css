.HomePage {
    font-family: var(--font-general);
    width: 100%;
}

/* SectionMenu Home*/

.SetionMenu {
    display: flex;
    flex-direction: column;
    padding-top: 50px;
}

/* Section Menu img Mobile */
.SectionMenu-img-movile {
    display: flex;
    justify-content: center;
}

/* Section Menu Text */

.SectionMenu-text{
    align-items: center;
    display: flex;
    flex-direction: column;
    height: 100%;
    justify-content: space-between;
}

.SectionMenu-text-container{
    text-align: center;
    margin-bottom: 140px;
    max-width: 200px;
}

.SectionMenu-text-container h1{
    font-size: var(--text-heading);
    font-weight: bold;
}

.SectionMenu-text-container h2{
    color: var(--text-color-dark);
    font-size: var(--text-size-lg);
}

/* Section Menu acknowledgments */

.SectionMenu-text-img {
    align-self: center;
    display: flex;
    height: 50px;
    width: max-content;
}

.SectionMenu-text-img img{
    width: 50px;
}

.SectionMenu-text-img img:nth-child(1){
    width: 130px;
}

/* Section Menu img Desktop */
.SectionMenu-img {
    display: none;
}

/* Menu Bar Separate */

.SectionMenu-Bar {
    align-items: center;
    display: flex;
    flex-wrap: nowrap;
    justify-content: space-between;
    width: 100%;
}

.SectionMenu-line{
    background-color: var(--color-line);
    height: 2px;
    width: 29%;
}

.SectionMenu-circle{
    border-radius: 100%;
    border: 2px solid var(--color-line);
    height: 12px;
    width: 12px;
}

/* Section Info */

.SectionInfo{
    display: flex;
    flex-wrap: wrap;
    justify-content: space-evenly;
    margin: 0 auto;
    padding: 100px 0 0px;
    width: 95%;
}

.SectionInfo-item{
    display: flex;
    flex-direction: column;
    justify-content: flex-start;
    margin: 10px 7px;
    padding-bottom: 20px;
    text-align: center;
    width: 170px;
}

.SectionInfo-item img{
    height: 160px;
}

.SectionInfo-item h3{
    margin: 10px 0;
}

.SectionInfo-item p{
    font-size: var(--text-size-md);
    padding: 0 10px;
}

/* Section About */

.SectionAbout {
    align-items: center;
    display: flex;
    flex-direction: column;
    justify-content: center;
    margin: 0 auto;
    padding: 100px 0 150px;
    text-align: center;
    width: 90%;
}

.SectionAbout-more {
    margin-bottom: 20px;
}

.SectionAbout-more h2{
    font-size: var(--text-size-xl);
    margin: 15px 0;
}

.SectionAbout-more p {
    font-size: var(--text-size-md);
    margin: 0 auto;
    max-width: 430px;
}

.SectionAbout-info-container {
    display: flex;
    flex-direction: column;
}

.SectionAbout-container-info{
    display: grid;
    font-size: var(--text-size-xs);
    grid-gap: 20px 0;
    grid-template-columns: 1fr 1fr;
    margin-top: 20px;
}

.SectionAbout-info-item {
    display: flex;
    flex-direction: column;
    justify-content: space-between;
    align-items: center;
}

.info-item-top {
    font-weight: bold;
}

.info-item-footer {
    font-weight: lighter;
    color: var(--text-color-light-dark);
    margin-top: 2px;
}

.info-item-date {
    font-size: var(--text-size-md);
}

.info-item-location {
    max-width: 200px;
}

.item-contact {
    display: flex;
    align-items: center;
    justify-content: center;
}

.item-contact p{
    margin-left: 5px;
}

.info-item-icon:hover {
    color: var(--stars-hover-color);
}


@media only screen and (min-width: 750px) {
    /* Section Menu */
    .SetionMenu{
        flex-direction: row;
        padding-top: 0px;
        justify-content: space-between;
        height: calc(100vh - 60px);
    }
    
    .SectionMenu-img-movile {
        display: none;
    }
    
    .SectionMenu-text{
        padding: 0 30px;
    }

    .SectionMenu-text-container {
        text-align: start;
    }

    .SectionMenu-text-container h1{
        margin: 50px 0;
    }

    .SectionMenu-img {
        display: flex;
        justify-content: flex-end;
    }
    .SectionMenu-img img{
        height: 100%;
        border-radius: 50px 0 0 ;
        object-fit: cover;
    }

    /* Section About */

    .SectionAbout {
        flex-direction: row;
    }

    .SectionAbout-img{
        width: 400px;
        margin-right: 30px;
    }
}
