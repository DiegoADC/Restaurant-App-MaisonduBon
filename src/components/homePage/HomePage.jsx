import StarIcon from '@mui/icons-material/Star'
import MailIcon from '@mui/icons-material/Mail'
import PhoneIcon from '@mui/icons-material/Phone'
import HomeMobile from '../../assets/images/fonds/home-movile.jpg'
import HomeDesktop from '../../assets/images/fonds/home.jpg'
import RelaisRecognition from '../../assets/images/logos/Home-realais.png'
import FiftyRecognition from '../../assets/images/logos/Home-50.png'
import CompanyRecognition from '../../assets/images/logos/Home-B.png'
import Recognition from '../../assets/images/more/recognition.png'
import Quality from '../../assets/images/more/Quality.png'
import Events from '../../assets/images/more/Events.png'
import SteakHouse from '../../assets/images/logos/Steak-logo.png'
import './HomePage.css'

export const HomePage = () => {
  return (
    <main className='HomePage'>
      <section className='SetionMenu'>

        <div className='SectionMenu-img-movile'>
          <img src={HomeMobile} alt='Corte de carne' />
        </div>

        <div className='SectionMenu-text'>

          <div className='SectionMenu-text-container'>
            <h1>La Maison du Bon</h1>
            <h2>Cuidamos con pasión y detalle cada platillo que elaboramos para ti</h2>
          </div>

          <div className='SectionMenu-text-img'>
            <img src={RelaisRecognition} alt='Relais & Chateaux MaisonDuBon' />
            <img src={FiftyRecognition} alt='50 Best restaurants' />
            <img src={CompanyRecognition} alt='B company certified' />
          </div>

        </div>

        <div className='SectionMenu-img'>
          <img src={HomeDesktop} alt='Mesa de cortes' />
        </div>

      </section>

      <div className='SectionMenu-Bar'>
        <span className='SectionMenu-line' />
        <span className='SectionMenu-circle' />
        <span className='SectionMenu-line' />
        <span className='SectionMenu-circle' />
        <span className='SectionMenu-line' />
      </div>

      <section className='SectionInfo'>

        <article className='SectionInfo-item'>
          <img src={Recognition} alt='Image Recognition' />
          <h3>Reconocimiento</h3>
          <p>Tenemos grandes reconocimientos de los mejores críticos y varios premios</p>
        </article>

        <article className='SectionInfo-item'>
          <img src={Quality} alt='Image Quality' />
          <h3>Calidad</h3>
          <p>Hacemos nuestros platillos con la mejor selección de productos calidad</p>
        </article>

        <article className='SectionInfo-item'>
          <img src={Events} alt='Image Events' />
          <h3>Eventos</h3>
          <p>Por celebración de 1er mes de apertura, tenemos nuestros precios a un 20% de descuento</p>
        </article>

      </section>

      <section className='SectionAbout'>

        <img src={SteakHouse} alt='Logo Steak House' className='SectionAbout-img' />

        <div className='SectionAbout-info-container'>

          <article className='SectionAbout-more'>
            <h2>Sobre Nosotros</h2>
            <p>Cocina Especializada en cortes, es un restaurante que reúne a una selección de cocinas dentro de un espacio diseñado para brindar una experiencia agradable a todos los sentidos.</p>
          </article>

          <article className='SectionAbout-container-info'>

            <div className='SectionAbout-info-item'>
              <p className='info-item-top info-item-date'>8 de agosto de 2022</p>
              <h3 className='info-item-footer'>Creación</h3>
            </div>

            <div className='SectionAbout-info-item'>
              <p className='info-item-top info-item-location'>Del Refugio 40, Barrio del Atache, 40250 Taxco, Gro.</p>
              <h3 className='info-item-footer'>Ubicación</h3>
            </div>

            <div className='SectionAbout-info-item'>
              <div className='info-item-top'>
                <StarIcon className='info-item-icon' sx={{ transition: 'color 0.3s', color: '#474747' }} />
                <StarIcon className='info-item-icon' sx={{ transition: 'color 0.3s', color: '#474747' }} />
                <StarIcon className='info-item-icon' sx={{ transition: 'color 0.3s', color: '#474747' }} />
                <StarIcon className='info-item-icon' sx={{ transition: 'color 0.3s', color: '#474747' }} />
                <StarIcon className='info-item-icon' sx={{ transition: 'color 0.3s', color: '#474747' }} />
              </div>
              <h3 className='info-item-footer'>Rating</h3>
            </div>

            <div className='SectionAbout-info-item'>
              <div className='info-item-top'>
                <div className='item-contact'>
                  <MailIcon sx={{ color: '#929292' }} />
                  <p>MaisonContact@gmail.com</p>
                </div>
                <div className='item-contact'>
                  <PhoneIcon sx={{ color: '#929292' }} />
                  <p>+52 55 8725 2956</p>
                </div>
              </div>
              <h3 className='info-item-footer'>Contacto</h3>
            </div>

          </article>

        </div>

      </section>
    </main>
  )
}
