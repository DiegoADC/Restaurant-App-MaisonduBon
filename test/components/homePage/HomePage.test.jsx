import { render } from '@testing-library/react'
import { HomePage } from '../../../src/components/homePage/HomePage'

describe('Pruebas en <HomePage />', () => {
  test('debe hacer match con el SnapShot', () => {
    const component = render(<HomePage />)

    expect(component).toMatchSnapshot()
  })
})
