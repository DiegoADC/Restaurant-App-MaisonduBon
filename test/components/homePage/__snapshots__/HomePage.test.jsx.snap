// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HomePage /> debe hacer match con el SnapShot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <main
        class="HomePage"
      >
        <section
          class="SetionMenu"
        >
          <div
            class="SectionMenu-img-movile"
          >
            <img
              alt="Corte de carne"
              src=""
            />
          </div>
          <div
            class="SectionMenu-text"
          >
            <div
              class="SectionMenu-text-container"
            >
              <h1>
                La Maison du Bon
              </h1>
              <h2>
                Cuidamos con pasión y detalle cada platillo que elaboramos para ti
              </h2>
            </div>
            <div
              class="SectionMenu-text-img"
            >
              <img
                alt="Relais & Chateaux MaisonDuBon"
                src=""
              />
              <img
                alt="50 Best restaurants"
                src=""
              />
              <img
                alt="B company certified"
                src=""
              />
            </div>
          </div>
          <div
            class="SectionMenu-img"
          >
            <img
              alt="Mesa de cortes"
              src=""
            />
          </div>
        </section>
        <div
          class="SectionMenu-Bar"
        >
          <span
            class="SectionMenu-line"
          />
          <span
            class="SectionMenu-circle"
          />
          <span
            class="SectionMenu-line"
          />
          <span
            class="SectionMenu-circle"
          />
          <span
            class="SectionMenu-line"
          />
        </div>
        <section
          class="SectionInfo"
        >
          <article
            class="SectionInfo-item"
          >
            <img
              alt="Image Recognition"
              src=""
            />
            <h3>
              Reconocimiento
            </h3>
            <p>
              Tenemos grandes reconocimientos de los mejores críticos y varios premios
            </p>
          </article>
          <article
            class="SectionInfo-item"
          >
            <img
              alt="Image Quality"
              src=""
            />
            <h3>
              Calidad
            </h3>
            <p>
              Hacemos nuestros platillos con la mejor selección de productos calidad
            </p>
          </article>
          <article
            class="SectionInfo-item"
          >
            <img
              alt="Image Events"
              src=""
            />
            <h3>
              Eventos
            </h3>
            <p>
              Por celebración de 1er mes de apertura, tenemos nuestros precios a un 20% de descuento
            </p>
          </article>
        </section>
        <section
          class="SectionAbout"
        >
          <img
            alt="Logo Steak House"
            class="SectionAbout-img"
            src=""
          />
          <div
            class="SectionAbout-info-container"
          >
            <article
              class="SectionAbout-more"
            >
              <h2>
                Sobre Nosotros
              </h2>
              <p>
                Cocina Especializada en cortes, es un restaurante que reúne a una selección de cocinas dentro de un espacio diseñado para brindar una experiencia agradable a todos los sentidos.
              </p>
            </article>
            <article
              class="SectionAbout-container-info"
            >
              <div
                class="SectionAbout-info-item"
              >
                <p
                  class="info-item-top info-item-date"
                >
                  8 de agosto de 2022
                </p>
                <h3
                  class="info-item-footer"
                >
                  Creación
                </h3>
              </div>
              <div
                class="SectionAbout-info-item"
              >
                <p
                  class="info-item-top info-item-location"
                >
                  Del Refugio 40, Barrio del Atache, 40250 Taxco, Gro.
                </p>
                <h3
                  class="info-item-footer"
                >
                  Ubicación
                </h3>
              </div>
              <div
                class="SectionAbout-info-item"
              >
                <div
                  class="info-item-top"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium info-item-icon css-19sbr9r-MuiSvgIcon-root"
                    data-testid="StarIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                    />
                  </svg>
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium info-item-icon css-19sbr9r-MuiSvgIcon-root"
                    data-testid="StarIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                    />
                  </svg>
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium info-item-icon css-19sbr9r-MuiSvgIcon-root"
                    data-testid="StarIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                    />
                  </svg>
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium info-item-icon css-19sbr9r-MuiSvgIcon-root"
                    data-testid="StarIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                    />
                  </svg>
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium info-item-icon css-19sbr9r-MuiSvgIcon-root"
                    data-testid="StarIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                    />
                  </svg>
                </div>
                <h3
                  class="info-item-footer"
                >
                  Rating
                </h3>
              </div>
              <div
                class="SectionAbout-info-item"
              >
                <div
                  class="info-item-top"
                >
                  <div
                    class="item-contact"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1h3wrv4-MuiSvgIcon-root"
                      data-testid="MailIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 4-8 5-8-5V6l8 5 8-5v2z"
                      />
                    </svg>
                    <p>
                      MaisonContact@gmail.com
                    </p>
                  </div>
                  <div
                    class="item-contact"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1h3wrv4-MuiSvgIcon-root"
                      data-testid="PhoneIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M6.62 10.79c1.44 2.83 3.76 5.14 6.59 6.59l2.2-2.2c.27-.27.67-.36 1.02-.24 1.12.37 2.33.57 3.57.57.55 0 1 .45 1 1V20c0 .55-.45 1-1 1-9.39 0-17-7.61-17-17 0-.55.45-1 1-1h3.5c.55 0 1 .45 1 1 0 1.25.2 2.45.57 3.57.11.35.03.74-.25 1.02l-2.2 2.2z"
                      />
                    </svg>
                    <p>
                      +52 55 8725 2956
                    </p>
                  </div>
                </div>
                <h3
                  class="info-item-footer"
                >
                  Contacto
                </h3>
              </div>
            </article>
          </div>
        </section>
      </main>
    </div>
  </body>,
  "container": <div>
    <main
      class="HomePage"
    >
      <section
        class="SetionMenu"
      >
        <div
          class="SectionMenu-img-movile"
        >
          <img
            alt="Corte de carne"
            src=""
          />
        </div>
        <div
          class="SectionMenu-text"
        >
          <div
            class="SectionMenu-text-container"
          >
            <h1>
              La Maison du Bon
            </h1>
            <h2>
              Cuidamos con pasión y detalle cada platillo que elaboramos para ti
            </h2>
          </div>
          <div
            class="SectionMenu-text-img"
          >
            <img
              alt="Relais & Chateaux MaisonDuBon"
              src=""
            />
            <img
              alt="50 Best restaurants"
              src=""
            />
            <img
              alt="B company certified"
              src=""
            />
          </div>
        </div>
        <div
          class="SectionMenu-img"
        >
          <img
            alt="Mesa de cortes"
            src=""
          />
        </div>
      </section>
      <div
        class="SectionMenu-Bar"
      >
        <span
          class="SectionMenu-line"
        />
        <span
          class="SectionMenu-circle"
        />
        <span
          class="SectionMenu-line"
        />
        <span
          class="SectionMenu-circle"
        />
        <span
          class="SectionMenu-line"
        />
      </div>
      <section
        class="SectionInfo"
      >
        <article
          class="SectionInfo-item"
        >
          <img
            alt="Image Recognition"
            src=""
          />
          <h3>
            Reconocimiento
          </h3>
          <p>
            Tenemos grandes reconocimientos de los mejores críticos y varios premios
          </p>
        </article>
        <article
          class="SectionInfo-item"
        >
          <img
            alt="Image Quality"
            src=""
          />
          <h3>
            Calidad
          </h3>
          <p>
            Hacemos nuestros platillos con la mejor selección de productos calidad
          </p>
        </article>
        <article
          class="SectionInfo-item"
        >
          <img
            alt="Image Events"
            src=""
          />
          <h3>
            Eventos
          </h3>
          <p>
            Por celebración de 1er mes de apertura, tenemos nuestros precios a un 20% de descuento
          </p>
        </article>
      </section>
      <section
        class="SectionAbout"
      >
        <img
          alt="Logo Steak House"
          class="SectionAbout-img"
          src=""
        />
        <div
          class="SectionAbout-info-container"
        >
          <article
            class="SectionAbout-more"
          >
            <h2>
              Sobre Nosotros
            </h2>
            <p>
              Cocina Especializada en cortes, es un restaurante que reúne a una selección de cocinas dentro de un espacio diseñado para brindar una experiencia agradable a todos los sentidos.
            </p>
          </article>
          <article
            class="SectionAbout-container-info"
          >
            <div
              class="SectionAbout-info-item"
            >
              <p
                class="info-item-top info-item-date"
              >
                8 de agosto de 2022
              </p>
              <h3
                class="info-item-footer"
              >
                Creación
              </h3>
            </div>
            <div
              class="SectionAbout-info-item"
            >
              <p
                class="info-item-top info-item-location"
              >
                Del Refugio 40, Barrio del Atache, 40250 Taxco, Gro.
              </p>
              <h3
                class="info-item-footer"
              >
                Ubicación
              </h3>
            </div>
            <div
              class="SectionAbout-info-item"
            >
              <div
                class="info-item-top"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium info-item-icon css-19sbr9r-MuiSvgIcon-root"
                  data-testid="StarIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium info-item-icon css-19sbr9r-MuiSvgIcon-root"
                  data-testid="StarIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium info-item-icon css-19sbr9r-MuiSvgIcon-root"
                  data-testid="StarIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium info-item-icon css-19sbr9r-MuiSvgIcon-root"
                  data-testid="StarIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium info-item-icon css-19sbr9r-MuiSvgIcon-root"
                  data-testid="StarIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </div>
              <h3
                class="info-item-footer"
              >
                Rating
              </h3>
            </div>
            <div
              class="SectionAbout-info-item"
            >
              <div
                class="info-item-top"
              >
                <div
                  class="item-contact"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1h3wrv4-MuiSvgIcon-root"
                    data-testid="MailIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 4-8 5-8-5V6l8 5 8-5v2z"
                    />
                  </svg>
                  <p>
                    MaisonContact@gmail.com
                  </p>
                </div>
                <div
                  class="item-contact"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1h3wrv4-MuiSvgIcon-root"
                    data-testid="PhoneIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M6.62 10.79c1.44 2.83 3.76 5.14 6.59 6.59l2.2-2.2c.27-.27.67-.36 1.02-.24 1.12.37 2.33.57 3.57.57.55 0 1 .45 1 1V20c0 .55-.45 1-1 1-9.39 0-17-7.61-17-17 0-.55.45-1 1-1h3.5c.55 0 1 .45 1 1 0 1.25.2 2.45.57 3.57.11.35.03.74-.25 1.02l-2.2 2.2z"
                    />
                  </svg>
                  <p>
                    +52 55 8725 2956
                  </p>
                </div>
              </div>
              <h3
                class="info-item-footer"
              >
                Contacto
              </h3>
            </div>
          </article>
        </div>
      </section>
    </main>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
